src-vue/src/utils/thunderstore/ThunderstoreModVersion.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface ThunderstoreModVersion {
+    description: string;
+    icon: string;
+    version_number: string;
+    download_url: string;
+    downloads: number;
+    date_created: string;
+}
```

Comment:
```suggestion
}

```

Trailing new line :)